Documentation/Signoffs/jiegwuatu42-web_wireless-signoff.md

```diff
@@ -1,4 +1,4 @@
-# Website Signoff
+# Web and Wireless Signoff
 ## FUNCTION OF THE SUBSYSTEM
 
 The Wireless acquisition of data helps effortlessly boost the quality of life. The functionality can be broken into a couple of parts: the server, storage, and user interface (UI). The server accepts data from the device and stores it for remote access from the user. The data will be stored on a server host machine. The UI is what the user sees when attempting to read the data collected from the device. The UI helps get the data to the user in an understandable manner.
```

Comment:
First sentence does not specify a function. Second sentence does not specify a function. Third and fourth - what is the difference between the server and the server host machine, if any? Fifth sentence does not specify a function. Last sentence could use some clarification.